Workshops/2024-07-10_LLM-Evaluation/README.md

```diff
@@ -0,0 +1,64 @@
+# TMLS - LLM Evaluation
+
+In this stream we'll cover various aspects of LLM evaluation including:
+
+* Traditional evaluation, human evaluation, evaluating LLMs
+* Evaluation metrics and considerations
+* Evaluation tool and platform comparison, including open-sourced alternatives
+* Hands-on end-to-end examples
+
+
+## Environment Setup & Installation
+
+1. Setup your environment
+
+If you use Conda:
+
+```bash
+conda create -n llm-evaluation python=3.10
+conda activate llm-evaluation
+pip install -r requirements.txt
+```
+
+If you use Poetry:
+
+```bash
+poetry shell
+poetry install
+```
+
+If you use pure Python:
+
+```bash
+python -m venv env
+source env/bin/activate
+pip install -r requirements.txt
+```
+
+2. Create private environment file (this will not be committed!)
+```
+cp .env-template .env
+```
+
+Add any necessary API keys there following the given format. Specifically, you will need a key for your LLM (if you're using an API), a key for [LangSmith](https://smith.langchain.com/) (optional; it's called `LANGCHAIN_API_KEY` in the config), and a key for [Langfuse](https://langfuse.com/) (optional). Both LangSmith and Langfuse offer free API keys which can be obtained on their respective websites by creating an account. You can see the section below for mroe details.
```

Comment:
Also I don't seen langfuse or langsmith being used in the demo notebooks? Do we need to talk about the API keys here in that case?